### DISCUSSION TITLE
Use of Jolpica F1 in R Package

### DISCUSSION THREAD

**Discussion by pbulsink** [jolpica/jolpica-f1#87] [Show and tell]
Hello

I'm one of the maintainers of the f1dataR package for R users. We have a friendly R frontend for both the FastF1 python package and Ergast API calls. 

I have a PR open (Scasanova/f1dataR#270) for migrating our package from Ergast to Jolpica, but wanted to give you a heads up before that's fully implemented. 

You'll find calls to the API have a useragent string of `f1dataR/[version]` for easy identification. The package has a hard throttle limit of 4 requests/second and will respect Retry-After if present in the HTTP error 429 header.

Note that everything in the package is cached on the user's computer for (at minimum) the current development session, up to 24 hours in temp files, or the user can cache to persistent file.

Please reach our and let us know if you have any questions, comments, or concerns!

Thanks,

**Comment by jolpica** [comment]
Hey, that sounds awesome feel free to switch it over.

Just bear in mind we are still in Alpha so things may not be 100% reliable. Also make sure to check out our documented differences to Ergast.
Lastly, the Rate limits may decrease in the future, we'll try implement the "Retry-After" header if we do, but can't guarantee it.

Thanks for the heads up!

**Reply by jolpica** [reply]
Also if you have any questions feel free to reach out. (we'll try not to take so long to respond next time!)
